app/frontend/src/components/Welcome.vue

```diff
@@ -90,9 +75,9 @@ export default {
 
 <style scoped>
 .v-card {
-  margin-bottom: 2rem;
+  margin-bottom: 3rem;
```

Comment:
Now that I think about it, I suppose this margin is mainly to pad for spacing between cards. However, I think there's also an app-wide issue with the footer bar where default content will get masked by it. We can revisit it later, but if this margin is to try and address the footer overflow, then we should be adding margin to a higher level component or even the application level itself instead of doing it in the card.